config/locales/devise.en.yml

```diff
@@ -7,7 +7,7 @@ en:
       send_instructions: "You will receive an email with instructions for how to confirm your email address in a few minutes."
       send_paranoid_instructions: "If your email address exists in our database, you will receive an email with instructions for how to confirm your email address in a few minutes."
     failure:
-      already_authenticated: "You are already signed in."
```

Comment:
this message was auto-appearing on certain pages where it didn't make sense (from Devise `require_no_authentication` method)